src/Orleans.Core/Configuration/Options/GrainServiceOptions.cs

```diff
@@ -1,10 +1,34 @@
-using System;
 using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Options;
 
 namespace Orleans.Hosting
 {
+    /// <summary>
+    /// Configuration of silo grain services.
+    /// </summary>
     public class GrainServiceOptions
     {
+        /// <summary>
+        /// List of grain services to initialize at startup.  List of full type name (string) and service id (short).
+        /// </summary>
         public List<KeyValuePair<string, short>> GrainServices { get; set; } = new List<KeyValuePair<string, short>>();
```

Comment:
Not for this PR, but it seems like these should be registered as a collection of services, maybe keyed (I don't know) in future. Sound right to you?

Reply:
Agreed.  Grain services pattern needs work.  I refactored it some when moving from ClusterConfig, but it needs more love.